### PR TITLE
Updated README (see comments)

### PR DESCRIPTION
**Pull request by Symphon-y**
- Added Sheilds.io
- Added href links to sections
- Added Overview Descriptions
- Updated Installation Instructions
- Updated Git Work Flow

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,102 @@
...
-- ### Components
-  - #### Overview
-  - #### Questions & Answers
-  - #### Ratings & Reviews
-- ### Git Work Flow
-- ### Installation
-  - #### Dependencies
-- ### Testing
+<p align="center">
+<a href="#overview">Overview</a> |
```
Comment (sbirvin1s) [inline README.md:21]: Nice links!

File: README.md [hunk 2]
```diff
@@ -139,60 +189,63 @@ git push origin a-second-new-branch // push to the feature branch you created
 
 ## Installation:
 
-How can another developer get your project up and running on their own? What dependencies are required? Are there environmental requirements? Be specific, and outline steps to take in order to get the project running.
+Getting your own copy of Project Atelia is easy!
```
Comment (sbirvin1s) [inline README.md:192]: minor spelling error. Should be "Project Atelier"

**Review by sbirvin1s — CHANGES_REQUESTED** [review]
Minor spelling error of Project Atelier that should be corrected.

Everything else looks good!

**Review by mayliang021 — APPROVED** [review]
Looks good!

**Review by sbirvin1s — APPROVED** [review]
Spelling error fixed and approved

**Pull request merged**